src/main/java/frc/robot/RobotContainer.java

```diff
@@ -47,26 +48,44 @@ public class RobotContainer {
 
     private void configureAutoCommands(){
         NamedCommands.registerCommands(Map.of(
-            "armLow", 
-                m_arm.pickUp(),
-            "groundSlam",
-                m_arm.groundSlam(),
-            "shootSequence", Commands.sequence(
-                Commands.parallel(
-                m_arm.straightShot(),
-                m_shooter.straightAutoShot()),
-                m_intake.shootNote(),
-                Commands.waitSeconds(.25),
-                m_shooter.pauseUpBeat()            
-            ),
-            "shootSequence2", Commands.sequence(
-                Commands.parallel(
-                m_arm.sideShoot(),
-                m_shooter.straightAutoShot()),
-                m_intake.shootNote(),
-                Commands.waitSeconds(.25),
-                m_shooter.pauseUpBeat() 
-            )
+            "intake",Commands.parallel(
+                    m_shooter.pauseUpBeat(),
+                    m_arm.pickUp(),
+                    m_intake.autoIntake()),
+            "armUp",Commands.parallel(
+                    m_arm.straightShot(),
+                    m_shooter.autoShoot()),
+            "shoot",Commands.sequence(
+                    m_intake.autoIntake(),
```

Comment:
This explains why we have a long delay when shooting after missing a note.  It waits the whole 2.5 second timeout before moving on.  We can change this to the instant command.